htdocs/main.inc.php

```diff
@@ -887,7 +887,7 @@ function analyseVarsForSqlAndScriptsInjection(&$var, $type, $stopcode = 1)
 		if (GETPOST("username", "alpha", $allowedmethodtopostusername)) {	// For posting the login form
 			$goontestloop = true;
 		}
-		if (GETPOST('openid_mode', 'alpha', 1)) {	// For openid_connect ?
+		if (GETPOST('openid_mode', 'alpha')) {	// For openid_connect ?
```

Comment:
Can you confirm we must remove the parameter ",1" here ?

Reply:
This is to allow POST, this is used on the form inside 'callback.php'

We want to prevent the argument to stay in the URL when a user doesn't change page after login. When a new OpenID authentication is needed and this parameter is still present in the URL this cause an error since it's awaiting the code parameter if this parameter is present and the user not logged.

Reply:
@eldy